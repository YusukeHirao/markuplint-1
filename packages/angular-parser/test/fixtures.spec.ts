import path from 'path'

import { exec } from 'markuplint'

test('fixtures', async () => {
  const resultInfos = await exec({
    files: path.resolve(__dirname, 'fixtures/*.html'),
  })
  for (const { filePath, results } of resultInfos) {
    expect(results).toMatchSnapshot(path.basename(filePath))
  }
})
