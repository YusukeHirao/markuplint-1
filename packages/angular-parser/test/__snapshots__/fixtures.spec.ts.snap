// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixtures: basic.html 1`] = `
Array [
  Object {
    "col": 1,
    "line": 1,
    "message": "Element is deprecated",
    "raw": "<font color=\\"red\\">",
    "ruleId": "deprecated-element",
    "severity": "error",
  },
]
`;
